activerecord/test/cases/associations/has_many_associations_test.rb

```diff
@@ -206,6 +206,11 @@ def test_create_from_association_should_respect_default_scope
 
     bulb = car.bulbs.create(:name => 'exotic')
     assert_equal 'exotic', bulb.name
+
+    # {frickinawesome: false} is same to default value of
+    # Bulb#frickinawesome.
```

Comment:
I think we can :scissors: this comment. Maybe we can add some code to make sure this test is testing the right thing?